handbook/company/product-groups.md

```diff
@@ -207,7 +208,21 @@ The product designer prepares proposed changes in the form of wireframes for thi
 #### Developing from wireframes
 Please read carefully and [pay special attention](https://fleetdm.com/handbook/company/why-this-way#why-do-we-use-a-wireframe-first-approach) to UI wireframes.
 
-Designs have usually gone through multiple rounds of revisions, but they could easily still be overlooking complexities or edge cases!  When you think you've discovered a blocker, communicate.  Leave a comment [mentioning the appropriate PM](https://fleetdm.com/handbook/company/development-groups) or ask for feedback at your next standup.  Then update this user story's estimation, wireframes, and "definition of done" to reflect your updated understanding. 
+Designs have usually gone through multiple rounds of revisions, but they could easily still be overlooking complexities or edge cases! When you think you've discovered a blocker, here's how to proceed:
+
+**For UX concerns...**
+
+At Fleet, we prioritize [iteration](https://fleetdm.com/handbook/company#results). So before raising the alarm, think through the following:
+
++ Would addressing this add design work and/or delay shipping the feature?
++ Will this hurt the first-time user experience if we ship as-is?
++ Is this change a "one-way door"?
+
+After these considerations, if you still think you've found a blocker, alert the [appropriate PM](https://fleetdm.com/handbook/company/development-groups) so that the user story can be brought back for [expedited drafting](https://fleetdm.com/handbook/product#expedited-drafting). Otherwise, this concern may be better suited to a [feature request](https://fleetdm.com/handbook/product#intake).
+
+**For technical (implementation) concerns...**
+
+Communicate. Leave a comment [mentioning the appropriate PM](https://fleetdm.com/handbook/company/development-groups) or ask for feedback at your next standup. Then update this user story's estimation, wireframes, and "definition of done" to reflect your updated understanding.
```

Comment:
- Up to the PM to update the user story
- I like "or ask for feedback at next standup" but I think better to leave it out, since leaving a comment is more async-friendly.
-"the appropriate PM" can link to the table of product groups (less confusing when you click it, given the page you're on.  Will feel less broken)
- Instead of "technical (implementation) concerns", isn't it "For implementation concerns"  (regardless if technical or not)
- the implementation concerns one could go on top to make it more readable (it's also more common)
- "For UX concerns" could be "For all other concerns..." and then it's listed last.